chart/kubeapps/README.md

```diff
@@ -178,6 +178,7 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
 | `dashboard.customAppViews`                        | Package names to signal a custom app view                                                  | `[]`                 |
 | `dashboard.customComponents`                      | Custom Form components injected into the BasicDeploymentForm                               | `""`                 |
 | `dashboard.remoteComponentsUrl`                   | Remote URL that can be used to load custom components vs loading from the local filesystem | `""`                 |
+| `dashboard.skipAvailablePackageDetails`           | Skip package details when they are not available                                           | `false`              |
```

Comment:
Fine as is, just confused me at first because it's not skipping when the details are not available, but rather, skipping when the readme is not available.

Reply:
Yep, it's kinda misleading... perhaps something like `skipPackageDetailsWhenNoReadme` ? WDYT?
I guess the name came from the bulk rename from Chart->AvailablePackage

Reply:
Yep, sounds good.

Reply:
Actually, the reason it stuck out to me was that I was wondering whether the original use-case might be more that you want to skip it regardless? Ie. always skipp package details, rather than it being dependent on whether the readme is populated. Don't remember but might be worth checking.

Reply:
You're totally right, I missed that context (just thought it was a matter of update the PR to include the latest changes):

> So yes, I'm keen that we don't land #3227 as is (ie. don't depend on the README being present or not), but rather do:
>> ... 
> >Add a config option, skipAvailablePackageDetails which defaults to false to keep the current behaviour, but when true, the tiles presented in the catalog link directly to the deploy form.

Let me implement this way straight away.

Reply:
Ah - great, we had recorded the discussion on the issue :) Excellent, thanks for the update!